### DISCUSSION TITLE
長庚研發AI超音波檢測系統 盼推廣嬰幼兒「髖關節發育不良」檢查

### DISCUSSION THREAD

**Discussion by 20010906** [1101-GS4719-C/Python-Community-GS4719-C#310] [General]
林口長庚醫院研發出，檢驗嬰幼兒髖關節發育的AI超音波系統，此系統甚至能以手持方式操作，讓檢查不是只有大醫院能進行，診所以及偏鄉地區也能協助檢查，使有髖關節發育不良的嬰幼兒及早治療，減少因延誤診斷所導致的後遺症。
[文章連結](https://health.ltn.com.tw/article/breakingnews/3756520)

**Comment by 109802503gloria** [comment]
能普及到偏鄉地區真的很重要，讓所有的新生兒都能接受完善的檢查

**Reply by 107602505** [reply]
看到個也想到前幾篇的聊天機器人Woebot同樣也是照顧到偏鄉地區的人，只能說AI的發展應用的層面真的相當的廣可以對各個頁有很大的幫助。
